src-electron/db/db-mapping.js

```diff
@@ -456,6 +456,7 @@ exports.map = {
       deviceIdentifier: x.DEVICE_IDENTIFIER,
       deviceId: x.DEVICE_IDENTIFIER,
       deviceVersion: x.DEVICE_VERSION,
+      deviceName: x.NAME,
```

Comment:
We should just expose both name and typename...  Is there a reason not to expose both?